Tests/demistomock/demistomock.py

```diff
@@ -741,6 +741,20 @@ def incidents(incidents=None):
         )
 
 
+def incident(incident=None):
```

Comment:
the func does not take any argument and behaves the same in both scripts and integrations - it returns the incident it is ran in

Reply:
Sorry, my bad